doc/upgrading.md

```diff
@@ -2,6 +2,7 @@
 
 * If you use `verbose` (or `-v` argument), switch to `log_filters` (or `RUST_LOG` environment variable).
   Please note that it allows to set per-module filters, but module naming is considered unstable.
+  If you have used `-vv` (the default setting), switch to `--log-filters INFO`.
```

Comment:
I think this should say "the value suggested in the documentation" instead of "the default setting" since the default was ERROR.